master/internal/webhooks/authz_iface.go

```diff
@@ -10,10 +10,11 @@ import (
 // WebhookAuthZ describes authz methods for experiments.
 type WebhookAuthZ interface {
 	// GET /api/v1/webhooks
+	CanGetWebhooks(ctx context.Context, curUser *model.User) (workspaceIDsWithPermsFilter []int32, serverError error)
```

Comment:
Should we name this `WebhooksAvailableToUser` to make the naming distinct from most of the boolean `CanDoXYZ` authz functions?